<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>589</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnFunc0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/0.JPG</normaloff>../image/0.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>173</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>196</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/1.JPG</normaloff>../image/1.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>189</width>
      <height>39</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>132</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/2.JPG</normaloff>../image/2.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>125</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>174</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/3.JPG</normaloff>../image/3.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>167</width>
      <height>39</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>253</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/4.JPG</normaloff>../image/4.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>246</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>145</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/5.JPG</normaloff>../image/5.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>138</width>
      <height>68</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>139</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/6.JPG</normaloff>../image/6.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>132</width>
      <height>68</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFunc7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>175</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../image/7.JPG</normaloff>../image/7.JPG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>169</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChsFunc">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choosen function:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumFunc">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbla">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>b:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblc">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>c:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAvalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBvalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCvalue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
